### PR TITLE
fix qemu_disk_img_snapshot

### PR DESCRIPTION
**Pull request by hjydxy**
No matter change_md5 or change_md5 == md5 will raise error.
So in my opinion, there should be "or" instead of "and".

### CODE REVIEW

**Comment by hjydxy** [comment]
ping...

File: qemu/tests/qemu_disk_img_snapshot.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def run(test, params, env):
     error.context("Step3. change tmp file before apply snapshot", logging.info)
     snapshot_test.start_vm(params)
     change_md5 = snapshot_test.save_file(t_file)
-    if not change_md5 and change_md5 == md5:
+    if not change_md5 or change_md5 == md5:
```
Comment (balamuruhans) [inline qemu/tests/qemu_disk_img_snapshot.py]: looks good.

File: qemu/tests/qemu_disk_img_snapshot.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def run(test, params, env):
...
         raise error.TestError("Fail to change tmp file.")
```
Comment (balamuruhans) [inline qemu/tests/qemu_disk_img_snapshot.py]: can you use `exceptions` from `avocado` instead of `error` and change elsewhere

**Comment by hjydxy** [comment]
@balamuruhans   Is that so?

**Comment by hjydxy** [comment]
@balamuruhans I'm newcomer and I will be appreciate if you give me some advise

**Comment by balamuruhans** [comment]
Sure, @hjydxy please attach job log of your test run.

**Comment by hjydxy** [comment]
screenshot from 2017-03-06 18-19-21

File: qemu/tests/qemu_disk_img_snapshot.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import logging
 from autotest.client.shared import error
```
Comment (balamuruhans) [inline qemu/tests/qemu_disk_img_snapshot.py]: unused import statement can be removed

Comment (hjydxy) [inline qemu/tests/qemu_disk_img_snapshot.py]: ok!

**Comment by hjydxy** [comment]
screenshot from 2017-03-06 19-01-34

**Comment by hjydxy** [comment]
Here is the latest outcome!

**Comment by hjydxy** [comment]
Thank you！！！

**Pull request merged**